py/vttest/mysql_db.py

```diff
@@ -35,3 +35,7 @@ def port(self):
 
   def unix_socket(self):
     raise NotImplementedError('MySqlDB is the base class.')
+
+  def config(self):
+    """Returns the json config to output."""
+    raise NotImplementedError('MySqlDB is the base class.')
```

Comment:
I was going to delete this class. I think this kind of stuff is not necessary in Python, but wanted to check with you.